### PR TITLE
Water page edits

### PR DESCRIPTION
**Pull request by rkelson**
.

### CODE REVIEW

**Comment by rkelson** [comment]
I think that is all for the Water pages for now

File: data/geoscience/aquifers/index.html [hunk 1]
```diff
@@ -8,30 +8,25 @@
...
 ---
 <figure class="caption caption--right">
   <img class="caption__image" src="{% link images/AquifersLarge.png %}" alt="Aquifers" />
   <figcaption class="caption__text">Aquifers</figcaption>
 </figure>
-{% include abstract.html download="https://drive.google.com/drive/folders/0ByStJjVZ7c7mVmZudmFKS0hIUFE"
+{% include abstract.html
 name="Aquifers"
 stewards="AGRC, USGS, and DEQ"
 abstract="The Aquifers data package contains the Recharge Discharge Areas and the Basin Fill Boundaries along the Wasatch Front and adjacent areas." %}
```
Comment (steveoh) [inline data/geoscience/aquifers/index.html:32]:
```suggestion
abstract="The Aquifers data contains the Recharge Discharge Areas and the Basin Fill Boundaries along the Wasatch Front and adjacent areas." %}
```

File: data/planning/water-related-land/index.html [hunk 2]
```diff
@@ -35,6 +31,10 @@ <h3 id="WaterRelatedLandUse">Water Related Land Use</h3>
     <p>The data produced by the water-related land use program are used for various planning purposes. Some of these include: determining cropland water use, evaluating irrigated land losses and conversion to urban uses, planning for new water development, estimating irrigated acreages for any area, and developing water budgets. Additionally, the data are utilized by many other state and federal agencies. Water-related land use can be used to depict Irrigation type (Irrigated, Sub-Irrigated, Non-Irrigated, Riparian, Urban, etc).</p>
     <p>See the accompanying FGDC metadata for description of extensive Land Use, Irrigation Type, and Vegetation Cover codes. There are no constraints or warranties with regard to the use of this dataset.</p>
     <p>{% include contact.html subject=page.title contact=site.data.contacts.water_related_land_use %}</p>
+    <h5>Related Resources</h5>
+    <ul class="dotless">
+      <li><a href="https://docs.google.com/a/utah.gov/folderview?id=0B8agagPrSa5xQmJxMjJHNm9vekU&usp=gmail">Historic Water Related Land Use Data</a></li>
```
Comment (steveoh) [inline data/planning/water-related-land/index.html:36]:
```suggestion
      <li><a href="https://docs.google.com/a/utah.gov/folderview?id=0B8agagPrSa5xQmJxMjJHNm9vekU">Historic Water Related Land Use Data</a></li>
```

File: data/water/lakes-rivers-dams/index.html [hunk 3]
```diff
@@ -41,14 +37,15 @@
...
+abstract="The Lakes, Rivers, Streams & Springs data package contains feature layers that are derived from the <a href='https://nhd.usgs.gov/data.html'>National Hydrography Dataset (NHD)</a>. The NHD is the surface water component of the National Map created by the United States Geological Survey (USGS). The NHD contains a feature-based database that interconnects and uniquely identifies the stream segments or reaches that make up the nation's surface water drainage system. NHD data was originally developed at 1:100,000-scale and exists at that scale for the whole country. NHD data in it's entirety can be downloaded from the <a href='https://www.usgs.gov/core-science-systems/ngp/national-hydrography/access-national-hydrography-products'>USGS</a>.</p>
```
Comment (steveoh) [inline data/water/lakes-rivers-dams/index.html:40]:
```suggestion
abstract="The Lakes, Rivers, Streams & Springs data contains feature layers that are derived from the <a href='https://nhd.usgs.gov/data.html'>National Hydrography Dataset (NHD)</a>. The NHD is the surface water component of the National Map created by the United States Geological Survey (USGS). The NHD contains a feature-based database that interconnects and uniquely identifies the stream segments or reaches that make up the nation's surface water drainage system. NHD data was originally developed at 1:100,000-scale and exists at that scale for the whole country. NHD data in it's entirety can be downloaded from the <a href='https://www.usgs.gov/core-science-systems/ngp/national-hydrography/access-national-hydrography-products'>USGS</a>.</p>
```

File: data/water/watersheds/index.html [hunk 4]
```diff
@@ -9,17 +8,17 @@
...
 ---
 <figure class="caption caption--right">
   <img class="caption__image" title="map" src="{% link images/WatershedAreas.png %}" alt="WatershedAreas" />
   <figcaption class="caption__text">Watershed sub-basins</figcaption>
 </figure>
-{% include abstract.html download="https://drive.google.com/drive/folders/0ByStJjVZ7c7mTjFaU0hXdXpOTDQ"
+{% include abstract.html
 name="Watershed Areas"
 stewards="AGRC, BLM"
 abstract="Water.Watersheds_Area represents the watershed areas throughout Utah." %}
```
Comment (steveoh) [inline data/water/watersheds/index.html:24]:
```suggestion
abstract="Watershed areas represent the watershed area boundaries throughout Utah." %}
```

**Review by steveoh — COMMENTED** [review]
a few more changes requested

**Review by steveoh — APPROVED** [review]

**Pull request merged**